app/src/main/java/com/esoxjem/movieguide/listing/MoviesListingFragment.java

```diff
@@ -166,6 +166,12 @@ public void onDestroy() {
         ((BaseApplication) getActivity().getApplication()).releaseListingComponent();
     }
 
+    @Override
+    public void onResume() {
+        super.onResume();
+        initLayoutReferences();
```

Comment:
The `movies` object used inside `initLayoutReferences()` could be empty if the Activity is recreated, right?